### PR TITLE
Moving web-assembly Client to Blazor server side Client

### PR DESCRIPTION
**Pull request by jmc171144**

### CODE REVIEW

File: src/Trakx.IndiceManager.Client/Pages/_Host.cshtml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@{
+    Layout = null;
+}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BlazorApp2</title>
```
Comment (monsieurleberre) [inline src/Trakx.IndiceManager.Client/Pages/_Host.cshtml:13]:
```suggestion
    <title>Index Manager</title>

```

File: src/Trakx.IndiceManager.Client/Trakx.IndiceManager.Client.csproj [hunk 2]
```diff
@@ -1,26 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <RazorLangVersion>3.0</RazorLangVersion>
-    <ServiceWorkerAssetsManifest>service-worker-assets.js</ServiceWorkerAssetsManifest>
-  </PropertyGroup>
-  <Import Project="../Common.Projects.props" />
```
Comment (monsieurleberre) [inline src/Trakx.IndiceManager.Client/Trakx.IndiceManager.Client.csproj:8]: don't forget this one :)

File: src/Trakx.IndiceManager.Client/Trakx.IndiceManager.Client.csproj [hunk 3]
```diff
@@ -1,26 +1,13 @@
...
-    <ServiceWorker Include="wwwroot\service-worker.js" PublishedContent="wwwroot\service-worker.published.js" />
-  </ItemGroup>
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
```
Comment (monsieurleberre) [inline src/Trakx.IndiceManager.Client/Trakx.IndiceManager.Client.csproj:6]:
```suggestion

    <Import Project="../Common.Projects.props" />
    
```

File: src/Trakx.IndiceManager.Client/Pages/_Host.cshtml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <base href="~/" />
+    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
+    <link href="_content/Syncfusion.Blazor/styles/bootstrap4.css" rel="stylesheet" />
+    <link href="css/site.css" rel="stylesheet" />
```
Comment (jmc171144) [inline src/Trakx.IndiceManager.Client/Pages/_Host.cshtml:17]:
```suggestion
    <link href="css/site.css" rel="stylesheet" />
     <style>
    .sf-license-warning {
        visibility: hidden;
         }
    </style>
```

**Pull request merged**